docs/queryworkers/attribute-based-access.md

```diff
@@ -0,0 +1,260 @@
+---
+sidebar_position: 40
+title: Attribute-Based Access Control
+---
+
+For this example, assume the following organizations each have employees who must access their own data:
```

Comment:
We're missing the context that the queries are query workers, which are saved to create an API that uses attributes to grant or limit permissions based on the filtering in each.

Reply:
It's a bit long, but how is this?

"You can save multiple queries as query workers, and then run them concurrently to create an API that uses attributes to limit permissions for users and API keys."